### ISSUE TITLE
ingress controller 安装失败

### ISSUE THREAD

**Issue by rainbowechoes** [apache/apisix-helm-chart#496]
在内网自己搭建的 k8s 集群上，按照官网命令：
```shell
helm install apisix-ingress-controller apisix/apisix-ingress-controller --namespace ingress-apisix --create-namespace
```

pod 状态：
![image](https://user-images.githubusercontent.com/38370475/223083060-2e408d4f-ae62-4e80-b10e-18779f945700.png)

pod 信息：
![image](https://user-images.githubusercontent.com/38370475/223083146-03860d96-9c08-459e-b8a4-adca0ec541e1.png)

**Comment by jinjianming** [comment]
```
检查 charts/etcd/values.yaml 
persistence.storageClass是否为你环境所存在的storageClass。
```

**Comment by rainbowechoes** [comment]
> ```
> 检查 charts/etcd/values.yaml 
> persistence.storageClass是否为你环境所存在的storageClass。
> ```
改为了环境中的 storageClass，etcd 还是一样的错。但是多启动了一个 apisix pod

![image](https://user-images.githubusercontent.com/38370475/223121379-e6091be5-1e55-4523-8cf0-ee8aa97d3921.png)

这个 apisix 的信息为：
![image](https://user-images.githubusercontent.com/38370475/223121547-83afbcf4-3cb2-48e3-b4a4-dee626941897.png)

**Comment by rainbowechoes** [comment]
> The reason why APISIX is not running is that your ETCD cluster is still pending.
> 
> After you change the storage class, run the `kubectl describe` of the ETCD pods again and see why it cannot run.

hello, this is pods status:

![image](https://user-images.githubusercontent.com/38370475/223296055-4d611449-2879-4517-b1ec-45816c93da16.png)

use `kubectl describe pod apisix-etcd-0 -n apisix`, output like this:

![image](https://user-images.githubusercontent.com/38370475/223296246-c79a3dce-be95-4ea4-9c91-a8b3a8b32a3b.png)

**Comment by rainbowechoes** [comment]
没人处理吗

**Comment by tao12345666333** [comment]
Obviously, this is due to your cluster environment and there is nothing we can do about it.

You can learn more about Kubernetes volumes. https://kubernetes.io/docs/concepts/storage/volumes/
Or just search for the error. xref https://stackoverflow.com/questions/52668938/pod-has-unbound-persistentvolumeclaims

**Comment by rainbowechoes** [comment]
> Obviously, this is due to your cluster environment and there is nothing we can do about it.
> 
> You can learn more about Kubernetes volumes. https://kubernetes.io/docs/concepts/storage/volumes/ Or just search for the error. xref https://stackoverflow.com/questions/52668938/pod-has-unbound-persistentvolumeclaims

ok, I have deployed apisix、apisix dashboard、apisix ingress controller. 

but  I can't open any gateway of these, all of them return '{"error_msg":"404 Route Not Found"}'

![image](https://user-images.githubusercontent.com/38370475/224216310-ad95c545-10ac-441b-8241-918e342131ec.png)

**Comment by rainbowechoes** [comment]
I modify ipList to `0.0.0.0/0`, but same error

**Comment by Baoyuantop** [comment]
The etcd startup failure in an internal network environment is a deployment environment issue (image pulling, DNS resolution, etc.) rather than a chart defect. This issue has had no activity for over a year.

Closing as stale. If you are still experiencing this issue, please feel free to reopen or file a new issue with your environment details.
